### PR TITLE
 Add JavaScript DOM insertion to enable menu bar open/close functionality

### PR DESCRIPTION
**Pull request by Nessrine88**
Hello,

This pull request adds JavaScript functionality to the menu bar, allowing it to be toggled open and closed by clicking on a menu icon. When the menu icon is clicked, the menu list is displayed if it is currently hidden, and hidden if it is currently displayed.

Changes Made:
- Added event listener to the menu icon button
- Created JavaScript code to toggle the display of the menu list

Testing:
- Tested the functionality on different devices and browsers to ensure cross-compatibility
- Verified that the menu list opens and closes as expected
- Checked for any console errors or unexpected behavior

Please review this pull request. Thank you!

### CODE REVIEW

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @Nessrine88 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request 🔴 

You have submitted a project, that is missing the `.eslintrc.json` file

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -15,12 +15,18 @@
...
             <header class="toolbar ">
-                <a href="#" > <img class="menu-bar" src="images/Icon - Menu.png" alt="menu bar"></a>
-                <ul>
-                    <li><a href="#"> Protofolio</li></a>
-                    <li><a href="#"> About</li></a>
-                    <li><a href="#"> Contact</li></a>
+            
+              <a href="#">
+                <span id="menuIcon" class="menu-bar">&#9776;</span>
+                <span id="closeIcon" class="close-icon">&times;</span>
```
Comment (Shedrack-Sunday) [inline index.html]: - [x] Kindly change the color of your close button on the menu to that provided on the template 👍 . This is to show your skills in recreating a given design 👍 

###  Figma:
image

### Your work:
image

File: index.html [hunk 2]
```diff
@@ -15,12 +15,18 @@
...
+              </a>
+              
+                <ul id="menuList">
+                    <li><a href="#portofolio"> Portofolio</li></a>
+                    <li><a href="#about-me"> About</li></a>
+                    <li><a href="#contact-me"> Contact</li></a>
```
Comment (Shedrack-Sunday) [inline index.html]: - [x] From the design given, the menu covers the whole page in a white background when clicked. Kindly implement this 👍 
- [x] Kindly adjust the margin of the navigation items to the left a little to have the same dimensions as provided on the template 

### Template given:
image

### Your work:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @Nessrine88 and @jovasal    👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️  
- Nice work with the setup✔️  
  
# Required Changes  🌵 

- [ ] You made some impressive implementations which I must say is a job well done. However, there are a few  I suggest   you make In line with the requirement


_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 2 above)
Comment (usorfaitheloho) [reply index.html]: As suggested by the previous reviewer, your menu should cover the whole or entire screen in a white background. At the moment, your menu only covers a part of the screen and when you scroll down the menu part is gone .Please implement  the menu popup to be similar as it is with the design given for your Portfolio to look more presentable 😉 

Your design :

https://github.com/Nessrine88/Portfolio.github.io/assets/20537712/b45332ae-27fd-4c42-a5d5-58d6e28de4ec


Figma design :
image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @Nessrine88  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * Great job implementing some changes requested by the previous reviewer ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Nessrine88** [comment]
@usorfaitheloho
I apologize for the confusion, but I'm having difficulty understanding the specific changes that are required. Could you please provide more detailed instructions or clarify the modifications needed for the menu implementation?

**Comment by usorfaitheloho** [comment]
Sorry for replying now, I just saw your message. what I mean is that when a user scrolls down after clicking your mobile menu, he shouldn't see your home page but rather the menu popup background, this is what you have ⬇️ 

image

**Review by topeogunleye — APPROVED** [review]
### Hi @Nessrine88, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Made sure mobile menu covers the entire screen ✔️ 
 
## [Optional] suggestions:
- For future pull requests, please notice that your branch name `menu-with-js` does not follow the convention of being named after a feature or action. For example, you could name your branch `added-menu-with-js`. This makes it easier to track the changes and the purpose of each branch. 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**